attestation/src/bin/run.rs

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::Path;
+use std::process::Command;
+
+fn main() {
+    let wksp_root = Path::new("/home/jpenumak/enarx");
+    let keep = Path::new("/home/jpenumak/enarx/enarx-keep-sgx/target/./debug/enarx-keep-sgx");
```

Comment:
These should be relative paths

Reply:
@ambaxter Thanks!
I didn't notice this earlier.

Reply:
Looks good!